arwen/elrondapi/manBufOps.go

```diff
@@ -248,6 +274,7 @@ func v1_4_mBufferGetByteSlice(context unsafe.Pointer, sourceHandle int32, starti
 
 	gasToUse = math.MulUint64(metering.GasSchedule().BaseOperationCost.DataCopyPerByte, uint64(len(sourceBytes)))
 	metering.UseGas(gasToUse)
+	runtime.TraceGasUsed(mBufferGetByteSliceName, initialGasLeft-metering.GasLeft())
```

Comment:
why do you need initialGasLeft - metering.GasLeft - here you know the actual used gas by doing a single SUM.

Reply:
delete from all the places.